apps/mobile/src/screens/auth/components/sign-up-form/sign-up-form.tsx

```diff
@@ -48,12 +48,14 @@ const SignUpForm: React.FC<Properties> = ({ onSubmit }) => {
 	const handleFormSubmit = useCallback((): void => {
 		void handleSubmit((signUpSubmissionData: UserSignUpFormDto) => {
 			const { password } = signUpSubmissionData;
-			const confirmPassword = watch(ConfirmPasswordCustomValidation.FIELD);
+			const confirmPassword = watch(
```

Comment:
Update this to use `getValues`, instead of `watch`

Reply:
![image](https://github.com/user-attachments/assets/f488bff8-5f5c-43cd-b289-b26a62d137b5)

Is there something wrong?

Reply:
Yes, check `update-password-form.tsx`